src/text/non_blocking/raw_text_reader.rs

```diff
@@ -988,16 +987,9 @@ mod reader_tests {
         assert_eq!(reader.read_i64()?, 2);
         match reader.next() {
             // the failure occurs after reading the \n after 3, so it is identified on line 3.
-            Err(IonError::Incomplete {
-                position:
-                    Position {
-                        line_column: Some((line, column)),
-                        ..
-                    },
-                ..
-            }) => {
-                assert_eq!(line, 2);
-                assert_eq!(column, 0);
+            Err(IonError::Incomplete(e)) => {
+                assert_eq!(e.position().line(), Some(2));
+                assert_eq!(e.position().column(), Some(0));
```

Comment:
🗺️ This is a good example of the updated application code for inspecting an `IonError` that has cropped up.